packages/react-app/src/services/bounty.service.ts

```diff
@@ -305,7 +306,19 @@ export const getBounties = async (
 		...paginationOptions,
 	};
 
-	return await Bounty.aggregateFn(bountyQuery);
+	const aggregation = await Bounty.aggregateFn(bountyQuery);
+
+	// Populate the payee data from the User DB. Needed until wallet addresses are stored directly in bounties
+	// Had to use a for loop so we didn't end up with a bunch of parallel db connections
```

Comment:
Seems like you could await each operation before launching a new one?

Reply:
I found many instances of people trying that, and it doesn't seem to work. This handles it OK